hkm/static/hkm/vendor

```diff
@@ -0,0 +1 @@
+../../../bower_components/
```

Comment:
It's better to use django-bower's static files finder for this. See django-bower's installation docs on how to enable it.